Makefile

```diff
@@ -213,7 +218,7 @@ RABBITMQ_INTEGRATION_HOST_CONTAINER=rm-rabbitmq
 
 rabbitmq-background:
 	@echo "Starting RabbitMQ in background"
-	-docker run --rm --hostname rabbitmq -p 5672:5672 -p 15672:15672 -e RABBITMQ_DEFAULT_USER=lunar -e RABBITMQ_DEFAULT_PASS=lunar --name ${RABBITMQ_INTEGRATION_HOST_CONTAINER} -d rabbitmq:3-management
+	-bash -c "docker run --rm --hostname rabbitmq -p 5672:5672 -p 15672:15672 -e RABBITMQ_DEFAULT_USER=lunar -e RABBITMQ_DEFAULT_PASS=lunar --name ${RABBITMQ_INTEGRATION_HOST_CONTAINER} -d rabbitmq:3-management || true"
```

Comment:
I had problems with it failing, because the rabbitmq was already there, when I had integrations tests failing. So as far as i can see, this solves it by continuing even with problems starting rabbitmq

Reply:
The leading `-` will ignore the exit code of `docker run` as to continue execution of the target, so I can't see how this should help with that?
https://www2.ipp.mpg.de/~dpc/gmake/make_47.html

Reply:
You seem to be right. Don't know what I saw. Reversed the change.